### ISSUE TITLE
[code][web] PR view is empty on first load

### ISSUE THREAD

**Issue by akosyakov** [gitpod-io/gitpod#7263]
### Bug description

To reproduce:
- start a repo on PR
- a PR view will pop up, but it does not have content
- in output view for PR extension you will see something:
```
[Info] Failed to parse patch for outdated comments: Failed to execute git {
  "exitCode": 128,
  "gitCommand": "diff",
  "stdout": "",
  "stderr": "fatal: Invalid symmetric difference expression 120872bfce3bbb7a7d1b3372282166745d9f9ec2...013a41b886e9fee130b5780b94fabf2b31a7de46\n"
}
```
- click refresh in PR view, content will pop up now

I wonder whether it is race with unshallowing. If it is so we can hold on triggering preloading PR view till unshallow is done by supervisor or trigger refresh automatically.

### Steps to reproduce

See above.

**Comment by jeanp413** [comment]
I tested this in codespaces and the PR entensions works just fine while the repo is shallow so it's something else (you get a notification error but the changes still show in the view I assume they used github API instead of git commands)
![image](https://user-images.githubusercontent.com/25115070/146475300-993dd869-5d01-475b-8c1b-6770864290ea.png)

**Comment by akosyakov** [comment]
@jeanp413 Could it be that extension is outdated or too new?

**Comment by jeanp413** [comment]
version is the same as GHPR extension does github releases, they just released a new version `0.34.2`, will wait for openvsx to pick it up and test again

**Comment by jeanp413** [comment]
tested with version 0.34.2 and issue persist
